doc_src/Readme.md

```diff
@@ -106,44 +118,48 @@ The ones available in `doc_src` are only there for reference.
 
 ### macOS
 
-All the dependencies can be installed through MacPorts except for Saxon 6-5-5:
+`ant` can be installed with the Xcode tools:
+    `$ xcode-select --install`
+	
+All the other dependencies can be installed through [MacPorts](https://www.macports.org/) except for saxon and whc.
+    `$ sudo port install docbook-xsl-nons, docbook-xml-4.5 @5.0_1, fop @1.1_1, libxml2 @2.9.12_1`
```

Comment:
1. You don't need backticks around indented code blocks
2. None of the other CLI commands in this document are prefixed with the shell prompt; we should at 
3. The syntax of the `port` command is wrong:
  - You can't suffix the port names with commas
  - Specifying the version is meaningless at best because MacPorts only lets you install the current port version
4. Does `ant` still come with Xcode? Where is it on your system? I don't have it except for through MacPorts.

    I suspect it hasn't been bundled with Xcode for a long time. We should just instruct people to get it from MacPorts.